guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md

```diff
@@ -11,28 +11,28 @@ functional_areas:
 
 A backup can fail for any of the following reasons:
 
-*	<a href="#trouble-backup-space">Insufficient disk space</a>
-*	<a href="#trouble-backup-os">An operating system error</a>
-*	<a href="#trouble-backup-all">Backup fails</a>
+*	[Insufficient disk space](#trouble-backup-space)
+*	[An operating system error](#trouble-backup-os)
+*	[Backup fails](#trouble-backup-all)
 
 ### Insufficient disk space   {#trouble-backup-space}
 
 If the backup failed because of insufficient disk space, you should typically free up disk space by moving some files to another storage device or drive. However, there might be other ways to resolve the issue. See one of the following resources for tips:
 
-*	<a href="http://www.cyberciti.biz/datacenter/linux-unix-bsd-osx-cannot-write-to-hard-disk" target="_blank">8 Tips to Solve Linux & Unix Systems Hard Disk Problems Like Disk Full Or Can’t Write to the Disk</a>
-*	<a href="http://serverfault.com/questions/315181/df-says-disk-is-full-but-it-is-not" target="_blank">serverfault: df says disk is full, but it is not</a>
-*	<a href="http://unix.stackexchange.com/questions/125429/tracking-down-where-disk-space-has-gone-on-linux" target="_blank">unix.stackexchange.com: Tracking down where disk space has gone on Linux? </a>
+*	[8 Tips to Solve Linux & Unix Systems Hard Disk Problems Like Disk Full Or Can’t Write to the Disk](http://www.cyberciti.biz/datacenter/linux-unix-bsd-osx-cannot-write-to-hard-disk){: target="_blank"}
+*	[serverfault: df says disk is full, but it is not](http://serverfault.com/questions/315181/df-says-disk-is-full-but-it-is-not){: target="_blank"}
+*	[unix.stackexchange.com: Tracking down where disk space has gone on Linux? ](http://unix.stackexchange.com/questions/125429/tracking-down-where-disk-space-has-gone-on-linux){: target="_blank"}
 
 ### Operating system error   {#trouble-backup-os}
 
 Unfortunately, we can't recommend anything specific because of the variety of errors you might encounter. We can suggest, however, you:
 
 *	Contact your system administrator
-*	Search public forums like <a href="http://unix.stackexchange.com" target="_blank">stackexchange</a> or <a href="http://stackoverflow.com" target="_blank">stackoverflow</a>
-*	Open a <a href="https://github.com/magento/magento2/issues" target="_blank">GitHub issue</a> and we'll try to help
+*	Search public forums like [stackexchange](http://unix.stackexchange.com){: target="_blank"} or [stackoverflow](http://stackoverflow.com){: target="_blank"}
```

Comment:
Should be `StackExchange`, and I don't think there is supposed to be a space between `:` and `target` here.

Reply:
Also should be `Stack Overflow`